teams/compiler-contributors.toml

```diff
@@ -57,3 +57,12 @@ extra-teams = ["compiler"]
 name = "Compiler team contributors"
 repo = "http://github.com/rust-lang/compiler-team"
 description = "Contributing to the Rust compiler on a regular basis"
+
+[[zulip-groups]]
+name = "T-compiler"
+
+[[zulip-groups]]
+name = "T-compiler/meeting"
+
+[[zulip-groups]]
+name = "T-compiler/contrib-private"
```

Comment:
not sure this is needed, as contributors are a subteam of compiler, but I hope it is, otherwise types members automatically get access to the private stream, too